package.json

```diff
@@ -23,19 +24,17 @@
 		"url": "https://github.com/overture-stack/arranger.git"
 	},
 	"scripts": {
-		"all": "npm run start --modules",
 		"components:watch": "npm run watch --w modules/components",
-		"watch": "npm run watch --ws",
-		"publish::ci": "npm publish --ws --if-present",
+		"modules:build": "npm run build --w modules",
+		"modules:tag": "npm version --ws --force-publish --yes",
+		"modules:watch": "npm run watch --w modules",
 		"nuke:nodeModules": "find . -name 'node_modules' -type d -prune -exec rm -rf '{}' +",
-		"prepare": "npm run build:production --w modules",
```

Comment:
the main offender, Jenkins will now rely directly on running the `modules:build` script